### PR TITLE
Added flowchart to describe step taking after operation start

### PR DESCRIPTION
**Pull request by MathoMathiasCamara**

### CODE REVIEW

File: chapter-3-basics-II.md [hunk 1]
```diff
@@ -58,8 +58,13 @@ Now MORYX knows, which cell uses which color, but it still doesn't know which co
...
 
-Parameters are the connection between *Tasks* and *Activities*. The task, which
-is equivalent to the workplan step, can be parameterized and parameters can
+Parameters are the connection between *Tasks* and *Activities*. 
+In the production process it looks like this:
+
+![Flow chart](./chapter-3/flowChart.drawio.png)
+
+As you can see the resource will only get the *Acvitiy*. 
+The *Task*, which is equivalent to the *workplan step*, can be parameterized and parameters can
```
Comment (1nf0rmagician) [inline chapter-3-basics-II.md:67]: Does it make sense to have this mentioned in the diagram as well? I.e. "Is there a next workplan step/Task"

**Comment by 1nf0rmagician** [comment]
And in the diagram:
Take step -> Take Step
matching capability -> matching Capability
done Working -> done working

This way only MORYX terminology is in capital letters. One could even think about putting the terms in italic font to highlight even more which is a MORYX term and which is not 🤔

**Comment by MarisaGoergen** [comment]
@MathoMathiasCamara Could you please add the drawio file to the repository

**Comment by 1nf0rmagician** [comment]
> @MathoMathiasCamara Could you please add the drawio file to the repository

Or do we want to try the markdown based chart tool here for the first time? Looks like a quite simple diagram and then we would not need to have the draw.io file in the repo 🤔

**Comment by MarisaGoergen** [comment]
Sounds good

File: chapter-3-basics-II.md [hunk 2]
```diff
@@ -58,8 +58,12 @@ Now MORYX knows, which cell uses which color, but it still doesn't know which co
...
+As you can see the resource will only get the *Acvitiy*.
```
Comment (MarisaGoergen) [inline chapter-3-basics-II.md:66]: Please add new line

**Comment by 1nf0rmagician** [comment]
I like the mermaid diagram. The purpose of it was to not have the draw.io and ong file in the repo so please make sure to rebase and leave them out before we merge 😊

**Review by MarisaGoergen — APPROVED** [review]

**Pull request merged**